Stewarded codebases linked from home page (#57)
publiccodenet/publiccode.net: codebases/index.md

# Where we work: our codebases

The Foundation for Public Code helps to effectively scale codebases. We do this by supporting ambitious communities with [codebase stewardship](../codebase-stewardship/index.html).

Codebases are organized by [codebase maturity](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html).

## Assessment

In this phase, codebase communities and the Foundation for Public Code decide if:

* the codebase is a good fit for Foundation stewardship
* Foundation stewardship will meet the community's goals and ambitions

These codebases are currently in public assessment:

> * [Signalen](https://github.com/Amsterdam/signals): receiving, automatically classifying and prioritising residents’ nuisance reports

Reviewer: I am not sure, but should we perhaps link this to signalen.org instead?

Reviewer 2: If we wish to highlight the strong role of Amsterdam, perhaps link to "Signalen" at signalen.org with text like "based on SIA" linked to Amsterdam/signals?

Author: Good spot, thank you!

> # Where we work: our codebases

Reviewer 3: I think the word _where_ is confusing in this sentence, it keeps confusing me and makes me think of physical locations. For example I keep thinking, 'Dutch municipalities'.

```suggestion
# What we work on: our codebases
```

Author: I've removed the 5 Ws as being too contrived, after getting that feedback from multiple people.

> The Foundation for Public Code supports ambitious communities and their codebases to scale effectively, including smoother collaboration and shared governance.

Reviewer 4: ```suggestion
The Foundation for Public Code helps to effectively scale codebases. We do this by supporting ambitious communities to build shared governance practices and smooth collaboration around their codebases.
```

> Codebases are organized by [codebase maturity](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html).

Reviewer: I suggest reordering them in the order:

- Mature
- Incubation
- Assessment
- Attic

My reasoning is that we want to put the spotlight on those who are mature, and not de-incentivise those who are in early phases to move down the list.

Author: Interesting! I suggest we implement this only once we have mature codebases, instead of putting an empty category at the top right now.

> Codebases are organized by [codebase maturity](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html).

Reviewer 4: ```suggestion
We organize codebases into different phases according to their [maturity](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html).
```

Author: I'm not going to merge your change. This is because:
- I'd like to avoid adding extra words above the fold - our goal is for people to jump as quickly as possible into looking at the codebases
- We already have further changes planned to this page as part of #64

## Assessment

In this phase, codebase communities and the Foundation for Public Code decide if:

* the codebase is a good fit for Foundation stewardship
* Foundation stewardship will meet the community's goals and ambitions

These codebases are currently in public assessment:

* [Signalen](https://github.com/Amsterdam/signals): receiving, automatically classifying and prioritising residents’ nuisance reports

## Incubation

Codebases in incubation are working to comply with the [Standard for Public Code](https://standard.publiccode.net/).

These codebases are currently in incubation:

* [OpenZaak](https://github.com/open-zaak/open-zaak): backend component to help municipalities manage and share data internally

## Mature

Mature codebases and communities fully meet the Standard for Public Code.

There are currently no mature codebases under Foundation stewardship.

## Attic

Codebases and communitities that were once Standard compliant but aren't any more are in the attic.

> There are currently no codebases in our attic.

Reviewer 3: I'd say this is not where we do our work, and also is not relevant yet. i don't see what this section adds.

Author: I had included so that the lifecycle model was complete (for educational purposes). I've cut it now because:
- you're right that we don't anticipate anyone caring about this for a long time
- there are more valuable things we'd like people to pay attention to

Author: Update: we're adding this section back in in #111. @angelplasma reckons that nothing being in the attic is important to show as well - that we can be proud that nothing once Standard compliant has fallen out of usage, suffered a community collapse, or changed its values signficantly.
